sys/ztimer/Makefile.dep

```diff
@@ -84,7 +84,7 @@ endif
 # the counter is read, this propagates and leads to timing errors
 # on ztimer_msec that are higher than > +-1msec.
 # The same goes for the fe310 rtt.
-ifneq (,$(filter sam% fe310,$(CPU)))
+ifneq (,$(filter sam3 fe310,$(CPU)))
```

Comment:
BTW @dylad what `sam%` based BOARDs do you have?

Reply:
I have access to SAML10/SAML11, SAML21, a SAMD21 mini clone and a SAME54
I don't have any `sam3`.

Reply:
Why add sam3 when it wasn't there before?

Reply:
`sam3` uses a completely different driver, I wonder if it wasn't caught here by mistake.

Reply:
When I started this PR, this line was `ifneq (,$(filter sam% fe310,$(CPU)))`
Which include both sam0 and sam3.
With this PR, sam0 should work fine but since sam3 were disabled and no tests were run on it. I just leave it there.

Reply:
Should I remove it then ?

Reply:
I'd say so

Reply:
Note that there will be a mismatch with sam3 Kconfig since this feature is also disabled in sam3 Kconfig

Reply:
Should be fixed.